Documentation/RELEASE_NOTES.md

```diff
@@ -12,7 +12,41 @@
 
 ### Resolved Issues
 
+<<<<<<< HEAD
```

Comment:
Looks like this file has content from before the release. Please merge dev into your branch and then update this file with the information for this fix

Reply:
done